.github/workflows/diffs.yml

```diff
@@ -0,0 +1,346 @@
+            - name: Push new commit
+              uses: ad-m/github-push-action@master
+              with:
+                  github_token: ${{ secrets.MAGICIAN_TOKEN }}
+                  repository: modular-magician/ansible_collections_google
+                  branch: refs/heads/pr-${{ github.event.pull_request.number }}-new
+                  directory: /home/runner/work/magic-modules/ansible_collections_google
+                  force: true
+
+            - run: git reset --hard HEAD~
+            - run: bundle exec compiler -a -e ansible -o /home/runner/work/magic-modules/ansible_collections_google
+            - run: git add . && git commit -m "Previous diffbase."
+              working-directory: /home/runner/work/magic-modules/ansible_collections_google
+              continue-on-error: true
+            - name: Push old commit
+              uses: ad-m/github-push-action@master
+              with:
+                  github_token: ${{ secrets.MAGICIAN_TOKEN }}
+                  repository: modular-magician/ansible_collections_google
+                  branch: refs/heads/pr-${{ github.event.pull_request.number }}-old
+                  directory: /home/runner/work/magic-modules/ansible_collections_google
+                  force: true
+
+    tpg-test:
+        runs-on: ubuntu-latest
+        name: "Test TPG"
+        needs:
+            - tpg
+        steps:
+            - uses: actions/setup-go@v1
+              with:
+                  go-version: 1.11.x
+            - uses: actions/checkout@v1
+              with:
+                  path: go/src/github.com/terraform-providers/terraform-provider-google
+                  repository: modular-magician/terraform-provider-google
+                  ref: pr-${{ github.event.pull_request.number }}-new
+            - run: make lint && make test
+              working-directory: /home/runner/work/magic-modules/go/src/github.com/terraform-providers/terraform-provider-google
+
+    tpgb-test:
+        runs-on: ubuntu-latest
+        name: "Test TPGB"
+        needs:
+            - tpgb
+        steps:
+            - uses: actions/setup-go@v1
+              with:
+                  go-version: 1.11.x
+            - uses: actions/checkout@v1
+              with:
+                  path: go/src/github.com/terraform-providers/terraform-provider-google-beta
+                  repository: modular-magician/terraform-provider-google-beta
+                  ref: pr-${{ github.event.pull_request.number }}-new
+            - run: make lint && make test
+              working-directory: /home/runner/work/magic-modules/go/src/github.com/terraform-providers/terraform-provider-google-beta
+
+    post-comment:
+        runs-on: ubuntu-latest
+        name: "Generate comment"
+        needs:
+            - ansible
+            - inspec
+            - tpgb
+            - tpg
+            - tf-conversion
+        steps:
+            - name: Comment on PR.
+              uses: ndmckinley/comment-on-pr@master
```

Comment:
Can we stage this fork in `modular-magician` and add branch controls that require reviewers? I'm afraid of repeating what happened with Concourse, where we had a bunch of forks of tasks that basically only you'd touched.

Reply:
Extremely doable, but this is a very temporary container.  I will replace it very soon with a script that does not require a container.  We discussed briefly and it seems like it's okay to just open an issue for now, will do.